install-manager/src/main/java/cm/aptoide/pt/install_manager/repository/TaskInfoRepository.kt

```diff
@@ -26,10 +26,10 @@ interface TaskInfoRepository {
   suspend fun saveJob(taskInfo: TaskInfo)
 
   /**
-   * Removes all enqueued task jobs by package name.
+   * Removes enqueued task job by package name and timestamp.
```

Comment:
The `by...` part of the comment is irrelevant now. Because we remove the object as whole.

Reply:
Fixed